src/clj/harja/palvelin/raportointi/raportit/ymparisto.clj

```diff
@@ -108,15 +108,15 @@
              [{:otsikko "Määrä yhteensä" :leveys "8%" :fmt :numero :jos-tyhja "-"
                :excel [:summa-vasen (if urakoittain? 2 1)]}
               {:otsikko "Tot-%" :leveys "8%" :fmt :prosentti :jos-tyhja "-"}
-              {:otsikko "Maksimi\u00admäärä" :leveys "8%" :fmt :numero :jos-tyhja "-"}]))
+              {:otsikko "Suunniteltu määrä" :leveys "8%" :fmt :numero :jos-tyhja "-"}]))
 
       (mapcat
        (fn [[{:keys [urakka materiaali]} rivit]]
          (let [suunnitellut (keep :maara (filter #(nil? (:kk %)) rivit))
-               maksimi (when-not (empty? suunnitellut)
+               suunniteltu (when-not (empty? suunnitellut)
                          (reduce + suunnitellut))
                luokitellut (filter :luokka rivit)
-               materiaalirivit (filter #(not (:luokka %)) rivit)
+               materiaalirivit (remove #(nil? (:kk %)) rivit)
```

Comment:
Tuntuu jotenkin karulta tunnistaa materiaalirivit tämän tiedon perusteella, mutta ilmeisesti on ok, kun näin on muuallakin toimittu. Jos en olisi tehnyt tätä juuri ennen lomalle lähtöä, niin olisin varmaan koittanut miettiä vaihtoehtoja..